napalm/eos/eos.py

```diff
@@ -805,7 +806,15 @@ def default_neighbor_dict(local_as):
             neighbor_dict.update(
                 {"prefix_limit": {}, "local_as": local_as, "authentication_key": ""}
             )  # few more default values
-            return neighbor_dict
```

Comment:
I think the failure from https://travis-ci.org/github/napalm-automation/napalm/jobs/677076093#L1044-L1047 is related to this line - this internal function `default_neighbor_dict` must return something, but without this line, it returns None.

Reply:
Thanks @mirceaulinic . That was definitely one of the issues. But it's still failing. Will try and debug this weekend